reports/models.py

```diff
@@ -1 +1,29 @@
-# Create your models here.
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+
+
+@python_2_unicode_compatible
+class ReportTaskStatus(models.Model):
+    """ The status of the report generate tasks
+    """
+
+    STATUS_CHOICES = (
+        ('Pending', "Pending"),
+        ('Running', "Running"),
+        ('Sending', "Sending"),
+        ('Done', "Done")
```

Comment:
Usually for this case, we do something like
```
PENDING = 'P'
RUNNING = 'R'
....
STATUS_CHOICES = (
    (PENDING, 'Pending'),
...
)
```
This allows us to do three things. We can reference the value using `ReportTaskStatus.PENDING`, so we can immediately see whether we've set it to a valid value or not (Editor can autocomplete `status = ReportTaskStatus.PENDING`, but it cannot autocomplete `status = 'Pending'`), it allows us to use shorter values like `P` instead of `Pending`, but in our code we still know exactly what we mean, but take up less storage on the database server, and it ensures that we don't make mistakes like `if status == 'pending'`, which would be an easy mistake to make, and would result in the content of the if statement never being able to run.

Docs: https://docs.djangoproject.com/en/1.11/ref/models/fields/#choices